tests/integration/services/test_ruleset_activate.py

```diff
@@ -144,24 +145,10 @@ def get_activation_stats():
     }
 
 
-@pytest.mark.django_db
```

Comment:
Would this still be useful to have as a test using the default the restart policy?